include/socgraph.php

```diff
@@ -1244,18 +1244,20 @@ function poco_discover_federation() {
 			poco_check_server("https://".$server->host);
 	}
 
-	// Discover GNU Social Servers
-	if (!get_config('system','ostatus_disabled')) {
-		$serverdata = "http://gstools.org/api/get_open_instances/";
-
-		$result = z_fetch_url($serverdata);
-		if ($result["success"]) {
-			$servers = json_decode($result["body"]);
-
-			foreach($servers->data AS $server)
-				poco_check_server($server->instance_address);
-		}
-	}
+	// Currently disabled, since the service isn't available anymore.
```

Comment:
Multi-line changes should be commented out with `/* someObsoleteCode(); */` and not with single-line comments (// is for single-line comments only). This results in lesser changed lines and makes merges easier (as only a `/*` and a `*/` is being added).